.travis.yml

```diff
@@ -21,14 +21,15 @@ services: docker
 env:
   matrix:
   - INSTANCE=ubuntu-1404
-  - INSTANCE=ubuntu-1604
+  # - INSTANCE=ubuntu-1604
 
 before_script:
   - sudo iptables -L DOCKER || ( echo "DOCKER iptables chain missing" ; sudo iptables -N DOCKER )
   - eval "$(/opt/chefdk/bin/chef shell-init bash)"
   - /opt/chefdk/embedded/bin/chef --version
   - /opt/chefdk/embedded/bin/cookstyle --version
   - /opt/chefdk/embedded/bin/foodcritic --version
+  - /opt/chefdk/embedded/bin/chef gem install stove
```

Comment:
This was just to make lint tasks run on travis.ci

[here's why specifically :](https://travis-ci.org/maraaaa/chef-rocketchat/jobs/243889834#L294-L305)

```
0.73s$ /opt/chefdk/bin/chef exec rake
rake aborted!
LoadError: cannot load such file -- stove/rake_task
/home/travis/build/maraaaa/chef-rocketchat/Rakefile:5:in `<top (required)>'
Polyglot::PolyglotLoadError: Failed to load stove/rake_task using extensions rb, treetop, tt
/home/travis/build/maraaaa/chef-rocketchat/Rakefile:5:in `<top (required)>'
(See full trace by running task with --trace)
The command "/opt/chefdk/bin/chef exec rake" exited with 1.
Done. Your build exited with 1.
```

The Rakefile [requires "stove/rake_task"](https://github.com/maraaaa/chef-rocketchat/blob/master/Rakefile#L5), but stove is [only included for :development](https://github.com/maraaaa/chef-rocketchat/blob/master/Gemfile#L18).

Perhaps you can explain why travis isn't installing it as part of `bundle install`? And maybe suggest a better alternative than just brute forcing it on travis.... 

hmm, [well we're skipping our `bundle install`](https://github.com/maraaaa/chef-rocketchat/blob/master/.travis.yml#L12-L13)... so is doing it this way the best choice?